src/test/java/io/gravitee/policy/overriderequestmethod/OverrideRequestMethodPolicyIntegrationTest.java

```diff
@@ -39,24 +39,32 @@
  * @author GraviteeSource Team
  */
 @GatewayTest
-@DeployApi("/apis/override-http-method.json")
-class OverrideRequestMethodPolicyIntegrationTest
-    extends AbstractPolicyTest<OverrideRequestMethodPolicy, OverrideRequestMethodPolicyConfiguration> {
+@DeployApi("/apis/v3_definition/override-http-method.json")
+public class OverrideRequestMethodPolicyIntegrationTest
+    extends AbstractPolicyTest<OverrideRequestMethodPolicyV3, OverrideRequestMethodPolicyConfiguration> {
+
+    @Override
+    protected void configureGateway(GatewayConfigurationBuilder gatewayConfigurationBuilder) {
+        super.configureGateway(gatewayConfigurationBuilder);
+        gatewayConfigurationBuilder.set("api.jupiterMode.enabled", "true");
+        gatewayConfigurationBuilder.set("api.jupiterMode.default", "always");
+    }
 
     @ParameterizedTest(name = "Should change method {0} to GET")
     @ValueSource(strings = { "POST", "DELETE", "PATCH", "HEAD" })
-    void shouldChangeMethod(String method, WebClient client) {
+    void shouldChangeMethod(String method, HttpClient client) throws InterruptedException {
         wiremock.stubFor(get("/endpoint").willReturn(ok()));
 
-        TestObserver<HttpResponse<Buffer>> obs = client.request(HttpMethod.valueOf(method), "/test").rxSend().test();
-
-        awaitTerminalEvent(obs);
-        obs
-            .assertComplete()
-            .assertValue(response -> {
+        client
```

Comment:
Not big change but I would do .test() call before making the assertion. Something like that in pseudo code
client.rxRequest(HttpMethod.valueOf(method), "/test")
            .flatMap(request -> request.rxSend())
            .test()
            .assertValue(response -> {                 
                assertThat(response.statusCode()).isEqualTo(200);
            })
            .await()
            .assertComplete()
            .assertNoErrors();

Reply:
good catch